tests/Aql/ExecutorTestHelper.h

```diff
@@ -145,6 +151,13 @@ struct ExecutorTestHelper {
 
     SharedAqlItemBlockPtr expectedOutputBlock =
         buildBlock<outputColumns>(itemBlockManager, std::move(_output));
+
+    if (result == nullptr) {
+      EXPECT_EQ(expectedOutputBlock, nullptr);
+      EXPECT_EQ(_emptyOutput, true);
+      return;
+    }
```

Comment:
What happens if `result` is not `nullptr` but `_emptyOutput == true`. I think in that case `testOutputBlock` does a `nullptr`-access.